eng/Versions.props

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <UsingToolXliff>true</UsingToolXliff>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
-    <VersionPrefix>7.0.201</VersionPrefix>
+    <VersionPrefix>7.0.203</VersionPrefix>
```

Comment:
FYI, we released a 7.0.201 hotfix for the SDK and reversioned sdk/installer. We determined in tactics that we should advance templating ahead to match the current sdk/installer version. Hence the skip.